src/meta/src/barrier/state.rs

```diff
@@ -58,4 +68,27 @@ impl BarrierManagerState {
         self.in_flight_prev_epoch = next_epoch.clone();
         (prev_epoch, next_epoch)
     }
+
+    pub fn inflight_actor_infos(&self) -> InflightActorInfo {
+        self.inflight_actor_infos.clone()
+    }
+
+    // TODO: optimize it as incremental updates.
+    pub fn resolve_worker_nodes(&mut self, nodes: Vec<PbWorkerNode>) {
+        self.inflight_actor_infos.resolve_worker_nodes(nodes);
+    }
+
+    /// Before resolving the actors to be sent or collected, we should first add the newly
+    /// added actors into checkpoint control, so that these actor can be send and collect.
```

Comment:
```suggestion
    /// Before resolving the actors to be sent or collected, we should first add the newly
    /// added actors into inflight actor infos, so that these actor can be send and collect.
```

BTW, what about also documenting for `InflightActorInfo`?

Reply:
Sure, I will add some docs for them.